7.X/7.5.0/README.md

```diff
@@ -145,11 +145,11 @@ Use this method where the installation machine does not have network access to p
 1. Follow the Docker Log In to quay.io steps under the Access Requirements section.
 2. Pull the image containing the self-extracting tar:
   ```bash
-  docker pull quay.io/sysdig/installer:7.5.0-1-uber
+  docker pull quay.io/sysdig/installer:3.5.1-1-uber
```

Comment:
@darroyo-sysdig is this version of the uber file correct for this version of onprem?